docs/documentation/features/rules.md

```diff
@@ -318,7 +387,6 @@ Bundles that provide rules in json format should have the following folder struc
 
 ## Rule Templates
 
-*Please describe rule template approach*
 
 Rule templates can simplify the definition of rules with similar behavior, by providing additional configuration properties. Then rule instance definition only refers the rule template and provides the values of the configuration properties.
```

Comment:
make clear that a rule-template is a one-way template --> rules are not updated if the template changed